### PR TITLE
Cannot access site flow: update copy

### PR DESCRIPTION
**Pull request by keoshi**
#### Changes proposed in this Pull Request

* Update copy on troubleshoot option as per the suggestion on p1HpG7-9wA-p2#comment-40220
* Remove trailing dots on option titles.

#### Screenshots

Before | After
--|--
image | image

#### Testing instructions

* Spin up this PR.
* Go to `https://wordpress.com/settings/disconnect-site/SITE_URL?type=down`.
* Ensure the copy on the page looks good.

### CODE REVIEW

File: client/my-sites/site-settings/disconnect-site/down-flow.jsx [hunk 1]
```diff
@@ -57,7 +57,7 @@ function DownFlow( { confirmHref, backHref, site, recordTracksEvent: tracks } )
 					<FormattedHeader
 						isSecondary
 						align="left"
-						headerText={ translate( 'Confirm that your site loads.' ) }
+						headerText={ translate( 'Confirm that your site loads' ) }
```
Comment (a8ci18n) [inline client/my-sites/site-settings/disconnect-site/down-flow.jsx:60]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/my-sites/site-settings/disconnect-site/down-flow.jsx [hunk 2]
```diff
@@ -76,9 +76,9 @@ function DownFlow( { confirmHref, backHref, site, recordTracksEvent: tracks } )
 					<FormattedHeader
 						isSecondary
 						align="left"
-						headerText={ translate( 'Troubleshoot Jetpack.' ) }
+						headerText={ translate( 'Troubleshoot Jetpack' ) }
```
Comment (a8ci18n) [inline client/my-sites/site-settings/disconnect-site/down-flow.jsx:79]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/my-sites/site-settings/disconnect-site/down-flow.jsx [hunk 3]
```diff
@@ -76,9 +76,9 @@ function DownFlow( { confirmHref, backHref, site, recordTracksEvent: tracks } )
...
 						subHeaderText={ translate(
-							'If your site is working but you’re still seeing this error, let’s troubleshoot your Jetpack connection.'
+							'If your site is loading but you’re still seeing this error, this guide will help you troubleshoot the Jetpack connection.'
```
Comment (a8ci18n) [inline client/my-sites/site-settings/disconnect-site/down-flow.jsx:81]: :warning: This change will be queued for retranslation. We'll display the English text until then.

**Comment by folletto** [comment]
Copy changes look good! 👍

**Pull request merged**